proseco/fid.py

```diff
@@ -518,4 +521,22 @@ def get_fid_positions(detector, focus_offset, sim_offset):
 
     yang, zang = np.degrees(yfid) * 3600, np.degrees(zfid) * 3600
 
+    # Apply fid offset
```

Comment:
Comments should reflect *why* something is being done, not *what* is being done which is clear from the call to `get_fid_offset`.

Reply:
Which I suppose also speaks to deleting comments after CoPilot.